public/app/plugins/panel/xychart/XYChartPanel.tsx

```diff
@@ -184,11 +186,11 @@ export const XYChartPanel2 = (props: Props) => {
     }
 
     const legendStyle = {
-      flexStart: css`
-        div {
-          justify-content: flex-start !important;
-        }
-      `,
+      flexStart: css({
+        div: {
+          justifyContent: 'flex-start !important',
```

Comment:
does this !important work here? I had issues before with getting this to work with object css style 🤔

Reply:
just check it and `!important` doesn't make any difference here, so I removed it